files/style.css

```diff
@@ -0,0 +1,175 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2379;
+  position: absolute;
+  background-image: url("Ellipse 18.png");
+  background-repeat: no-repeat;
```

Comment:
**Note** Please do not close a PR except if it is approved.

- Please use the proper font size and font-family as it is in the Figma design, Currently, your font family is different from the Figma design 
Implement changes and request a review with the right PR